packages/core/src/__tests__/ceramic.test.ts

```diff
@@ -141,8 +141,8 @@ describe('Ceramic integration', () => {
         stream3,
         2000,
         (state) => StreamUtils.statesEqual(state, stream1.state),
-        () => {
-          throw new Error(`streamtype3.state should equal streamtype1.state`)
+        (state) => {
+          expect(state).toEqual(stream1.state)
```

Comment:
Can this actually be equal at some point? Why not use `statesEqual` as in the happy path if so?

Reply:
because this way gives a nicer error message.  The way it was written just tells you that they don't equal, the new way will print out the diff of what's actually different between the two